src/styles/index.scss

```
@@ -1,3 +1,125 @@
+@import "/src/styles/variables";
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
```

try to use ` margin-bottom` to add outer indent between elements
```suggestion
    margin-bottom: 40px;
```